Gemfile

```diff
@@ -2,9 +2,10 @@
 
 source 'https://rubygems.org'
 
-gem 'octocat', '= 0.0.2'
+gem 'octokit', '~> 4.6'
 gem 'libxml-ruby', '> 0'
 gem 'libxslt-ruby', '> 0'
+gem 'licensee', '~> 8.7' # CHK TODO this requires cmake and a bunch of gnarly dependencies, as the comment in README.md mentions, should we just patch octokit (hasn't been updated since 2014) to get this now that we can assume github provides this?
```

Comment:
I think we can ignore this comment for now, since it was an initial comment when we were trying to set this up on our local. And were thinking if this would really work when go ahead and deploy to heroku or something. I will remove this todo.